packages/components/pie-textarea/src/index.ts

```diff
@@ -33,6 +45,21 @@ export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
     @validPropertyValues(componentSelector, resizeModes, defaultProps.resize)
     public resize = defaultProps.resize;
 
+    @property({ type: String })
+    public autocomplete?: TextareaProps['autocomplete'];
+
+    @property({ type: Boolean })
+    public autoFocus?: TextareaProps['autoFocus'];
+
+    @property({ type: Boolean })
+    public readonly?: TextareaProps['readonly'];
+
+    @property({ type: String })
+    public defaultValue?: TextareaProps['defaultValue'];
+
+    @property({ type: Boolean })
+    public required?: TextareaProps['required'] = false;
```

Comment:
If there is a default value that can go in `defaultProps`

Reply:
I thought about it and seems that I don't need a separate `defaultValue` property because, the `value` property itself can be used to set the initial value.